src/ucp/api/ucp.h

```diff
@@ -160,9 +160,9 @@ enum ucp_feature {
     UCP_FEATURE_AM            = UCS_BIT(6),  
 
     /**
-     * Request selection of transports with capability to perform local
-     * operations using a peer's memory buffer and memory handle created
-     * with @ref UCP_MEM_MAP_EXPORTED capability.
+     * Request selection of transports with capability to support mapping a
+     * peer's memory handle created with @ref UCP_MEM_MAP_EXPORT memory mapping
```

Comment:
`Support mapping a peer's memory handle that was created by @ref ucp_mem_map with the flag @ref UCP_MEM_MAP_EXPORT and use it for local operations`

Reply:
done